docs/traefik.md

```diff
@@ -0,0 +1,104 @@
+---
+id: traefik
+title: traefik reverse proxy
+---
+
+# Traefik static config file
+
+/etc/traefik/traefik.yml
+```yaml
+api:
+  dashboard: true
+  insecure: true
+
+entryPoints:
```

Comment:
Without the certificateResolvers for cloudflare, this settings file is pretty small and should be configured as traefik cli arguments.

Reply:
ill fix this later